doc/source/regression/wp.md

```diff
@@ -44,7 +24,7 @@ kernelspec:
 import pywt
 ```
 
-This helper function that can format arrays in a consistent manner across
+This helper function can format arrays in a consistent manner across
```

Comment:
This helper function, `format_array()`, is strange because in both documents where it is defined I never actually see it called. Do you know what that's about?

Reply:
I notice It is also defined in a different way in a different notebook:

```python
def format_array(arr):
    return "[%s]" % ", ".join(["%.14f" % x for x in arr])
```

and it is called in places like this: `>>> print(format_array(wavelet.rec_lo), format_array(wavelet.rec_hi))` or this: 

```python
for n in wp.get_leaf_nodes():
    print(n.path, format_array(n.data))
```

but it is legacy code after all, so we should be able to drop it. I had kept it solely because it was not directly related to the rendering of the notebooks, but cleanups are always good to do.

Reply:
It's just odd that two of the regression docs start with defining this function but then neither of them call it. It made me wonder if there was some side effect happening, which is why I didn't edit it out.